features/attached_enable.feature

```diff
@@ -706,6 +706,81 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
            | focal   |
            | jammy   |
 
+    @series.xenial
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached enable livepatch
```

Comment:
Instead of manually manipulating the contents of the reboot-required files, can we get the system into "real" states that result in each reboot-required status?

I think this should be possible on a xenial vm with esm. I think there should be patches available to the kernel in apt in esm-infra.

So the system starts out as "no"
then enable esm-infra and livepatch and upgrade the kernel
now status should be "yes-kernel-livepatches-applied"
then install something else from esm-infra that requires a reboot - openssh-server?
now status should be "yes"